control/lidar/README.md

```diff
@@ -0,0 +1,5 @@
+Once you have this package in your catkin workspace succesfully, do this command:
```

Comment:
Removed the old line about needing to update the launch file as I think @wolfenstein11x or @Robin-Parmentier fixed this.